tests/test-sources/plugins/languages/otter.nix

```
@@ -1,38 +1,51 @@
 {
   empty = {
-    plugins.otter.enable = true;
+    plugins = {
+      otter.enable = true;
+      treesitter = {
+        enable = true;
+        settings.highlight.enable = true;
```

`plugins.treesitter.enable` is fine, but not needed to avoid the warning or to run the test.

```suggestion
        # Avoid the warning
        settings.highlight.enable = true;
```

---

Good point ! Done